packages/devtools_app/test/shared/common_widgets_test.dart

```diff
@@ -211,4 +211,53 @@ void main() {
       expect(findCheckboxValue(), isTrue);
     });
   });
+
+  group('AreaPaneHeader', () {
+    const titleText = 'The title';
+    const centerActionText = 'The Center Action';
+    const centerActionContainerKey = Key('scrollableCenterActionsContainer');
+    const centerAction = Text(centerActionText);
+
+    setUp(() {
+      setGlobal(IdeTheme, IdeTheme());
+    });
+
+    testWidgets('actions do not take up space when not present',
+        (WidgetTester tester) async {
+      await tester.pumpWidget(
+        wrap(
+          const AreaPaneHeader(
+            title: Text(titleText),
+          ),
+        ),
+      );
+
+      final Row row = tester.widget(find.byType(Row)) as Row;
+      expect(
```

Comment:
also check for the presence of titleText

Reply:
hehe I beat you to this one 😄 
just pushed it up